config/crd/bases/kuadrant.io_tlspolicies.yaml

```diff
@@ -0,0 +1,295 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.10.0
+  creationTimestamp: null
+  name: tlspolicies.kuadrant.io
+spec:
+  group: kuadrant.io
+  names:
+    kind: TLSPolicy
+    listKind: TLSPolicyList
+    plural: tlspolicies
+    singular: tlspolicy
+  scope: Namespaced
+  versions:
+  - additionalPrinterColumns:
+    - description: TLSPolicy ready.
+      jsonPath: .status.conditions[?(@.type=="Ready")].status
+      name: Ready
+      type: string
+    name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: TLSPolicy is the Schema for the tlspolicies API
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: TLSPolicySpec defines the desired state of TLSPolicy
+            properties:
+              certManager:
```

Comment:
Went back and forth on this one, we could inline it `CertManagerSpec json:",inline"`, but this would allow you to add specs for others as unlikely as that may be. I'm easy either way tbh, this just makes it a bit more obvious it's certManager we/users are dealing with.